Automattic-Tracks-iOSTests/Tests/Event Logging/EventLoggingUploadManagerTests.swift

```diff
@@ -0,0 +1,76 @@
+import XCTest
+@testable import AutomatticTracks
+
+class EventLoggingUploadManagerTests: XCTestCase {
```

Comment:
In my previous PR review I tried to explain my thought process for property usage, but it was hard to do so without examples, so I took the time to create one for this PR: f7d87b2b11f2be9fe17344b9ef23f8f980f4ffae.

Similar to my previous PRs it doesn't have much value in tests, but I hope that it'll at least present a good example of how to work with immutability in tests.

There are a couple other files with very minor possible improvements similar to these, but I am leaving these because it's much of the same thing and mentioning these from here after won't have any example value either.